docs/static_landing_page/index.html

```diff
@@ -27,30 +27,30 @@
       		<div class="collapse navbar-collapse" id="navbarResponsive">
 		        <ul class="navbar-nav pt-2 mt-1 mr-xl-auto ml-xl-2 pl-xl-1 text-center text-nowrap">
 				  <li class="nav-item px-1">
-					  <a class="nav-link" href=./v0.6.1/quickstart.html>Getting Started</a>
+					  <a class="nav-link" href=./v0.6.2/quickstart.html>Getting Started</a>
 				  </li>
 		          <li class="nav-item px-1">
 					  <!-- Versioned Link -->
-		            <a class="nav-link" href="./v0.6.1/user_guide/index.html">User Guide</a>
+		            <a class="nav-link" href="./v0.6.2/user_guide/index.html">User Guide</a>
 		          </li>
 		          <li class="nav-item px-1">
 					<!-- Versioned Link -->
-		            <a class="nav-link" href="./v0.6.1/api_reference/index.html">API Docs</a>
+		            <a class="nav-link" href="./v0.6.2/api_reference/index.html">API Docs</a>
 		          </li>
 				  <li class="nav-item px-1">
 					<!-- Versioned Link -->
-		            <a class="nav-link" href="./v0.6.1/auto_examples/index.html">Example Notebooks</a>
+		            <a class="nav-link" href="./v0.6.2/auto_examples/index.html">Example Notebooks</a>
 		          </li>
 		          <li class="nav-item px-1">
 		            <a class="nav-link" href="#contribute">Contributor Guide</a>
 		          </li>
 				  <li class="nav-item px-1">
 					<!-- Versioned Link -->
-		            <a class="nav-link" href="./v0.6.1/faq.html">FAQ</a>
+		            <a class="nav-link" href="./v0.6.2/faq.html">FAQ</a>
 		          </li>
 				  <li class="nav-item px-1">
 					<!-- Versioned Link -->
-		            <a class="nav-link" href="./v0.6.1/about/index.html">About Us</a>
+		            <a class="nav-link" href="./v0.6.2/about/index.html">About Us</a>
```

Comment:
are these duplicated in two PRs? Ideally they shouldn't

Reply:
Yes I followed the release instructions. There is some variation but also a little overlap

Reply:
With the current setup, you don't actually need to update the docs on the branch, only `__init__.py` and the `ReadMe.md`. The docs are only built from `main`.

Reply:
Right, I got unnecessarily nervous about that. :-)